vortex-expr/src/lib.rs

```diff
@@ -156,6 +156,45 @@ impl VortexExprExt for ExprRef {
     }
 }
 
+#[derive(Clone, Debug, Hash, PartialEq, Eq)]
+pub struct AccessPath {
```

Comment:
I'm glad you found this name. I've had a heckin' time thinking of a good name for this back when I added FIeldOrIdentity.